olive/passes/openvino/quantization.py

```diff
@@ -65,39 +82,81 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
                 type_=Union[DataConfig, Dict],
                 description="Data config for calibration, required if dataloader_func is None.",
             ),
-            "metric_func": PassConfigParam(
-                type_=Union[Callable, str],
+            "model_type": PassConfigParam(
+                type_=str,
```

Comment:
should enum be better than allowed_values?

Reply:
Do you mean defining allowed values as enum? `allowed_values` set is simpler than an enum with additional definition and validation.

Reply:
with enum you can only need definition. @jambayk, what's your thought?

Reply:
https://docs.pydantic.dev/2.0/usage/types/enums/

Reply:
Ah not aware that pydantic supports Enums here. This looks better than manual validation. Will update with Enum.